Pipeline/DeployUCDComponentVersion/ucd-deploy.groovy

```diff
@@ -0,0 +1,192 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
+import groovy.transform.*
+import groovy.json.JsonSlurper
+
+import java.security.KeyManagementException
+import java.security.NoSuchAlgorithmException
+import java.security.SecureRandom
+import java.security.cert.CertificateException
+import java.security.cert.X509Certificate
+
+import javax.net.ssl.HttpsURLConnection
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+
+import org.apache.http.impl.client.CloseableHttpClient
+import org.apache.http.impl.client.HttpClientBuilder
+import org.apache.http.client.methods.HttpPut
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory
```

Comment:
Can we clean-up the imports?

Reply:
We need these imports ->  unable to resolve class without